src/components/CodeEditor/CodeEditor copy.js

```diff
@@ -0,0 +1,369 @@
+/**
```

Comment:
Why is this file added?

Reply:
I was starting to translate this one before.
But, I'll fix this by removing this file.

Thank you